searchcore/src/tests/proton/documentdb/maintenancecontroller/maintenancecontroller_test.cpp

```diff
@@ -1054,74 +1003,6 @@ TEST_F("require that periodic session prunings are scheduled",
     ASSERT_TRUE(f._gsp.isInvoked);
 }
 
-TEST_F("require that active bucket is not moved until de-activated", MaintenanceControllerFixture)
```

Comment:
@baldersheim Is a similar test relevant for bucket mover v2?

Reply:
The bucket move job is tested thoroughly in documentbucketmover_test.cpp. So I think this one is not necessary.